### PR TITLE
chore: simplify Autostart explanations

### PR DESCRIPTION
**Pull request by khorne3**

### CODE REVIEW

File: environments/autostart.md [hunk 1]
```diff
@@ -6,26 +6,20 @@ description: Learn how to configure automated environment rebuilds.
 Coder [automatically turns off idle
 environments](../admin/environment-management/shutdown.md) to help manage
 resource expenditure. Typically, this means environments turn off overnight and
-remain offline until a rebuild is requested. With Autostart, rebuilds may be
-automatically requested at a time that best suits your workflow. You can expect
-your environments to be ready for you at the start of each work day.
+remain offline until a rebuild is requested. With Autostart, you can request
+rebuilds at a time that suits your workflow. You can expect your environments to
```
Comment (fuskovic) [inline environments/autostart.md:10]:
```suggestion
automated rebuilds at a time that suits your workflow. You can expect your environments to
```

File: environments/autostart.md [hunk 2]
```diff
@@ -6,26 +6,20 @@ description: Learn how to configure automated environment rebuilds.
...
+be ready for you at the start of each workday.
 
 ## Criteria for Autostart
 
-Your environment must meet the following criteria for you to use Autostart:
+Your environment must be
 
-1. You've [set the Autostart time and selected
-   environments](#enabling-autostart) to be started automatically at that time.
+- Active (e.g., has been opened within the last five days)
```
Comment (fuskovic) [inline environments/autostart.md:17]: If it hasn't been opened in 5 days, it won't trigger. 

Inactive means 5 days or more.

With that said, active would mean it's been opened in the last 4 days.

File: environments/autostart.md [hunk 3]
```diff
@@ -6,26 +6,20 @@ description: Learn how to configure automated environment rebuilds.
...
-1. Your environments are off at the Autostart time (Autostart won't work if your
-   environments are on to prevent the possibility of triggering a rebuild while
-   you're working).
-
-1. Your environment must be active. We define *inactive* environments as those
-   that haven't been opened within the last five days.
-
-> Coder may trigger Autostart up to five minutes before your scheduled time to ensure
-all queued environments are ready by their scheduled time.
+Coder may trigger Autostart before your scheduled time to ensure all queued
```
Comment (fuskovic) [inline environments/autostart.md:21]: Not sure why we removed the detail explaining how many minutes before here.

`before` alone is too broad imo

```suggestion
Coder may trigger Autostart up to 5 minutes before your scheduled time to ensure all queued
```

File: environments/autostart.md [hunk 4]
```diff
@@ -44,7 +38,7 @@ all queued environments are ready by their scheduled time.
 ### Enabling Autostart for New Environments
 
 When creating a new environment, you may enable Autostart by checking the box
-labeled **Automatically turn this environment on at (HH:MM)** (where HH:MM will
-display as your configured time).
+labeled **Automatically turn this environment on at (HH:MM)** (where HH:MM is
+your configured time).
```
Comment (fuskovic) [inline environments/autostart.md:42]: I like this. This sounds clearer.

**Review by fuskovic — CHANGES_REQUESTED** [review]

**Review by fuskovic — APPROVED** [review]

**Review by vapurrmaid — APPROVED** [review]
Much better, looks great!

**Pull request merged**